src/openzaak/management/commands/generate_swagger_component.py

```diff
@@ -30,8 +32,12 @@ def handle(
         urlconf,
         component=None,
         *args,
-        **options
+        **options,
     ):
+        _version = getattr(settings, f"{component.upper()}_API_VERSION")
+
+        # Setting must exist for vng-api-common, so monkeypatch it in
+        settings.API_VERSION = _version
```

Comment:
~In that case we can keep the placeholder setting in [src/openzaak/conf/includes/api.py](https://github.com/open-zaak/open-zaak/pull/457/files#diff-10f23095d64e39bd1017698c8afa68bfR4) that you `del`'d explicitly.~ Oh it's a management command...